### ISSUE TITLE
[BUG] Scanner Keeps getting Stuck :/

### ISSUE THREAD

**Issue by Fxnarji** [Andrewthe13th/Inventory_Kamera#283]
⬇⬇ Fill in the forms below where you can answer/applies to you. ⬇⬇

## **What happed? What is seems to be a bug?**
Please describe what seems to be an issue:

- For Characters, the Scanner gets lost and ends up opening the "Change Avatar Menu"
- For Weapons and Artifacts, the Scanner crashes after a few Weapons/Artifacts
- For Character Ascention mats, seems to work fine
- For Materials, skips to the bottom, scans a few materials in reverse then stops

For more Details look here:
https://www.youtube.com/watch?v=FPwfD44hxvA

## **How to reproduce**
Please describe how can WE reproduce the issue. Feel free to add your log folder (by zipping and dropping here) to speed up our processes:
Click "Scan Genshin"
## **Expected behavior**
Please describe what you expect from Inventory Kamera or output files:

## **Info about your device**
- Device Operating System : Windows 11
- Genshin Impact Version : OSRELWin2.6.0_R6708157_S7320343_D631353
- Inventory Kamera Version : V1.2.5
- Screen resolution affected : 1920x1080 Windowed
 
## **Checklist before sending issue**
Check the tickbox by replacing [ ] with [x] (in editor) or tick it after leaving the Text Editor:
- [x] I am using the latest version of Inventory Kamera or/and have clicked Update Lookup Tables to update the database.
- [x] I am using *English* as a game language
- [x] I am using the 16:9 or 16:10 window screen and leave the screen in the Paimon menu before starting the application.
- [x] I tried setting the delay to maximum and still experiencing issues.
- [x] I have correctly set my character and inventory keys in Inventory Kamera
- [x] I do NOT have any image filtering softwares enabled for Genshin (HDR, Image Sharpening, etc.)

If you cannot do any of the above, please describe why.

## **Additional Notes**
Add any other context about the problem and log files/report here.
[logging.zip](https://github.com/Andrewthe13th/Inventory_Kamera/files/8719030/logging.zip)

**Comment by Cupcak3** [comment]
Duplicate of #40 
There is a problem with ultrawide in that it compresses the UI if you change the in-game resolution from ultrawide to standard. Restarting the game will fix this. 

I do not have an ultrawide monitor so I cannot easily fix the issue.

**Comment by Fxnarji** [comment]
Yup, can confirm restarting the Game fixed everything. If you want to test IK on Ultrawide, i can give you feedback/screenshots for UI Layout etc
